src/grb_common.jl

```diff
@@ -53,6 +53,29 @@ macro grb_ccall(func, args...)
     end
 end
 
+macro grb_ccall_intercept(model, func, args...)
+    f = "GRB$(func)"
+    quote
+        ccall(:jl_exit_on_sigint, Void, (Cint,), convert(Cint,0))
+        ret = try
+            $(@windows? :(ccall(($f,libgurobi), stdcall, $(args...))) : :(ccall(($f,libgurobi), $(args...))) )
```

Comment:
Just tried it, it still solves normally. Sending ctrl-c inside cmd or powershell does nothing, and inside cygwin it segfaults.

Gurobi and JuMP tests are both fine FWIW